docs/manual/java/guide/devmode/ServiceLocator.md

```diff
@@ -77,4 +128,4 @@ In sbt:
 
 @[service-locator-disabled](code/build-service-locator.sbt)
 
-Be aware that by disabling the Service Locator your services will not be able to communicate. To restore communication, you will have to provide an implementation of [`ServiceLocator`](api/index.html?com/lightbend/lagom/javadsl/api/ServiceLocator.html) in your service.
+Be aware that by disabling the Service Locator your services will not be able to communicate. You will also be unable to access to your services from the outside unless you access the exact host:port. To restore communication, you will have to provide an implementation of [`ServiceLocator`](api/index.html?com/lightbend/lagom/javadsl/api/ServiceLocator.html) in your service.
```

Comment:
In addition, we might want to provide more details to describe how to provide an implementation of `ServiceLocator` in this situation, but maybe not as part of this PR.